example/ios/Podfile

```diff
@@ -1,10 +1,8 @@
 source 'https://cdn.cocoapods.org/'
```

Comment:
@kuhnroyal Do you think developers still have to add this source?

Reply:
I don't think so